src/der/parser.rs

```diff
@@ -303,7 +303,7 @@ pub fn der_read_element_content_as(
         | BerTag::GeneralString => {
             der_constraint_fail_if!(i, constructed);
         }
-        BerTag::UtcTime | BerTag::GeneralizedTime => match i.last() {
+        BerTag::UtcTime | BerTag::GeneralizedTime => match i.get(len - 1) {
```

Comment:
Is this safe against malicious zero length content? The `last()` can not panic, this one can if `len` is zero.

Reply:
Good catch, thanks.